cloud/aws/templates/aws_oidc/app.tf

```diff
@@ -0,0 +1,107 @@
+module "ecs_cluster" {
+  source = "cn-terraform/ecs-cluster/aws"
+  name   = var.app_prefix
```

Comment:
Can you add the tags block to these?  They all support it

Reply:
Done